scripts/build/build/targets.py

```diff
@@ -271,6 +271,8 @@ def HostTargets():
     builder.AppendVariant(name="asan", conflicts=['tsan'], use_asan=True),
     builder.AppendVariant(name="libfuzzer", requires=[
                           "clang"], use_libfuzzer=True),
+    builder.AppendVariant(name="coverage", requires=[
```

Comment:
Use coverage does not seem to be a valid argument - I would expect changes in host.py and maybe unit tests.

the shell script for coverage runs things using lcov. Should builders run the same thing when coverage is enabled for a host builder?

Reply:
This was left-over change from prototyping.  Removing.